app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
-  I think this is okay because, in the likes table, the column **user_id** exists. But I think you should have maintained the author_id to avoid future issues.